web/profiles/custom/yalesites_profile/config/sync/responsive_image.styles.image_content_width.yml

```diff
@@ -0,0 +1,45 @@
+uuid: b960773d-0a57-42b5-9597-79fefe948b93
+langcode: en
+status: true
+dependencies:
+  config:
+    - image.style.3_2_1120
+    - image.style.3_2_1280
+    - image.style.3_2_1440
+    - image.style.3_2_1600
+    - image.style.3_2_1760
+    - image.style.3_2_1920
+    - image.style.3_2_2080
+    - image.style.3_2_2240
+    - image.style.3_2_2400
+    - image.style.3_2_320
+    - image.style.3_2_480
+    - image.style.3_2_640
+    - image.style.3_2_800
+    - image.style.3_2_960
```

Comment:
Images that are just "content" shouldn't be cropped. So this should use the `max_width` image styles instead of the 3_2 ones. See /admin/config/media/responsive-image-style/full_width